README

```diff
@@ -140,7 +129,15 @@ PCR-04: 93 33 4E 81 A6 9C 80 54 D6 87 C7 FD 76 7C 6F 4C 70 FC C6 73
 
 ... then your TPM is supported by your kernel.
 
-If your tpm has already been owned in the past, you can reset it by running
+TPM2.0 introduced different hash algorithms to PCRs. Those can be read by:
+
+# cat /sys/class/tpm/tpm0/pcr-sha256/<N>
+
+... where <N> is PCR number, between 0 and 23 inclusive. Different algorithms
+may be used in place of 'sha256', but TPM vendors rarely implement anything
+beyond SHA1 and SHA256 required by the TPM specification.
+
+If your TPM has already been owned in the past, you can reset it by running
 tpm_clear -z, powering your computer off, and then resetting TPM in the BIOS
```

Comment:
not tpm2_clear?

Reply:
I don't have TPM2.0, can you test it? For TPM1.2 it is `tpm_clear -z`, for TPM2.0 it may be `tpm2_clear` with or without additional switches.

Reply:
The problem is that it won't work. You need the authentication of platform hierarchy for example. Otherwise TPM clear is only possible from BIOS setup level

Reply:
Interestingly, `tpm_clear` is not even present under Qubes. I'll just remove this part, then, BIOS reset should do this anyway.

Reply:
Changed in https://github.com/TrenchBoot/qubes-antievilmaid/compare/74a1e4d3162389dcabe65f3e6781e56238ba327f..c2b7a55520af9571a84c8ebbfe6565e94ea230b5